Framework/include/QualityControl/TrendingTaskConfig.h

```diff
@@ -61,6 +61,7 @@ struct TrendingTaskConfig : PostProcessingConfig {
 
   bool producePlotsOnUpdate{};
   bool resumeTrend{};
+  bool trendOnlyIfAllInputsAreAvailable{ false };
```

Comment:
I think you also have to read this config value from the provided configuration, see the .cxx